### PR TITLE
display grade override history error from lms

### PR DESCRIPTION
**Pull request by jansenk**
When fetching grade override history, check for new response values `success` and `error_message`, and if `success` is false, display `error_message` rather than the default error message.

Screen Shot 2020-09-01 at 4 04 30 PM

I'm not sure what's up with the spacing on the error message. It looks like the existing error message also looks kinda weird when it doesn't completely span the screen. I'll attach some more screenshots.

@edx/masters-devs-gta 

###  **Important!!!** 
this currently will only work on this branch!! (https://github.com/edx/edx-platform/pull/24849) At least, until the PR is merged

### CODE REVIEW

**Comment by jansenk** [comment]
Screen Shot 2020-09-01 at 4 10 58 PM

**Comment by jansenk** [comment]
It certainly squishes a bit weirdly. Probably a new ticket! 
Screen Shot 2020-09-01 at 5 02 52 PM

File: src/data/actions/grades.test.js [hunk 1]
```diff
@@ -176,4 +180,92 @@ describe('actions', () => {
...
+          errorMessage: responseData.error_message,
+        }];
+        const store = mockStore();
+
+        return store.dispatch(fetchGradeOverrideHistory(subsectionId, userId)).then(() => {
+          expect(store.getActions()).toEqual(expectedActions);
+        });
+      });
+
+      it('on 500 error', () => {
```
Comment (muselesscreator) [inline src/data/actions/grades.test.js:256]: nit: I would use `test('on 500 error')` or describe more in the title what the expected behavior is.

File: src/data/reducers/grades.test.js [hunk 2]
```diff
@@ -161,4 +162,17 @@ describe('grades reducer', () => {
       type: ERROR_FETCHING_GRADES,
     })).toEqual(expected);
   });
+
+  describe('updates fetch grade history failure state', () => {
```
Comment (nsprenkle) [inline src/data/reducers/grades.test.js:166]: I would keep this as an `it`:
```suggestion
it('updates fetch grade history failure state', () => {
```

Comment (jansenk) [reply src/data/reducers/grades.test.js:166]: good catch, this is left over from something else i tried

**Review by nsprenkle — APPROVED** [review]
Have verified correct error banner behavior.

**Pull request merged**